spec/support/black_hole.rb

```diff
@@ -2,9 +2,11 @@
 
 module BlackHole
   class << self
+    # rubocop:disable Style/MethodMissingSuper
```

Comment:
You should be able to move this down as a suffix of the `def` and remove the `enable` comment afterwards:

```ruby
def method_missing(*) # rubocop:disable Style/MethodMissingSuper
```